### DISCUSSION TITLE
Enable PlayBook after installation

### DISCUSSION THREAD

**Discussion by JAEP2** [Security-Onion-Solutions/securityonion#7903] [General]
Hello Sirs,

Is possible to enable Playbook, if previusly has not been selected during the installtion?

I just  added a node with IDH, and I want to have the alerts of IDH in the alerts panel, but during the installation I did not installed playbook.

Will be apreciated to know if is possible to add the playbook to the SOC tools.

I try to put from 0 to 1 in /opt/so/saltstack/local/pillar/minions/$manager.sls

manager:
  mainip: 'XX.XX.XX.XX'
  mainint: 'eth0'
  proxy: ''
  no_proxy: ''
  esheap: '5421m'
  esclustername: '{{ grains.host }}'
  freq: 0
  domainstats: 0
  mtu:
  elastalert: 1
  es_port: 9200
  grafana: 1
  osquery: 0
  **playbook: 1**

Appear now during so-status, but finally i get and error.

Thanks you in advanced

**Comment by cm-ops** [comment]
Run the following command as well - `so-playbook-reset` It will takes some time to import the rules.

**Reply by JAEP2** [reply]
@cm-ops  Thanks a lot!  Works fine!!!
